cristin-commons/src/main/java/no/unit/nva/cristin/model/CristinQuery.java

```diff
@@ -74,6 +78,17 @@ public URI toNvaURI() {
             .getUri();
     }
 
+    /**
+     * Builds identifier URI in NVA format with added query and facet parameters.
+     */
+    public URI toNvaFacetURI() {
```

Comment:
facets in only supported in the new version. but if facets are supplied in version 1, and they are not valid in that version, it will throw exception probably? we might get away by including everything in the original method since parameters can only be passed that are valid for that version?